apps/order/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from django.db.models import F
+
+from apps.order.models import CartItem
+from config.serializers import CreatedUpdatedBaseSerializer
+
+
+class CartItemSerializer(CreatedUpdatedBaseSerializer, serializers.ModelSerializer):
+
+    class Meta:
+        model = CartItem
+        fields = ('book', 'quantity',)
+
+    def create(self, validated_data):
+        created_by = self.context['request'].user
+        validated_data['created_by'] = created_by
```

Comment:
Isn't this handled by CreatedUpdatedBaseSerializer?

Reply:
@thenav56  In serializer if we don't have create, update methods it inherits form CreatedUpdatedBaseSerializer, but if we have create, update methods in child serializer it overrides parent create and update so, we have handle this in child serializer

Reply:
why are we using create for both create/update? I think the client should fetch this and do an update instead of sending create.

Reply:
@frozenhelium  Can this is handed in client side?

Reply:
@thenav56  @frozenhelium  I changed this logic